test/test_jit_fuser_te.py

```diff
@@ -1288,7 +1288,6 @@ def apply(fn):
             return lambda x, y: fn(x, y)
 
         dtypes = [
-            # FIXME: Fails in IR Eval: torch.int8 and_ cpu
```

Comment:
This FIXME is for a different issue.

Reply:
What's the issue?  It looked like a vestigial FIXME.

Reply:
The issue is that the combination op=and, dtype=int8, device=cpu fails on IR eval :) I added it in one of the recent commits

Reply:
Ohhh, so what threw me off was this being in the block of `dtypes`.  I read it as saying the combination of int8 and cpu fails :-p.  Anyways I'll put it back